config_builder/materializer.py

```diff
@@ -45,8 +49,31 @@ def materialize_file(
     materialized_path = root_dir / materialized_root / relative_path
     os.makedirs(materialized_path.parent, exist_ok=True)
 
+    def _import_callback(module: Path):
+        if module.is_file():
+            content = module.read_text()
+        elif module.exists():
+            raise RuntimeError("Attempted to import a directory")
```

Comment:
I think this will also catch symlinks. Please check if that is the case: whether symlinks return `true` or `false` when calling `is_file`